src/RageUtil.cpp

```diff
@@ -33,6 +33,16 @@ void UnicodeUpperLower(wchar_t *, std::size_t, const unsigned char *);
 
 RandomGen g_RandomNumberGenerator;
 
+int fastround(const float num)

```

Comment:
> Also I have a pretty strong preference to just inline the cast instead of needing this function call. Introducing a separate commit that just does the std::lrint -> static_cast should be sufficient imo.

I'm happy to do this. Would you prefer `static_cast<int>(std::floor())` to `static_cast<int>()` in order to maintain function closer to `std::lrint`?

> can you enlighten me on what the impact would on rounding vs truncating in this PR?

It is typically being called on a non-integer value in the places where I've changed it, it seems that it would not affect these computations.

From my testing the two seem to provide the same result but `static_cast<int>()` is faster. The "tweakB" build that went around for a while had all `std::lrint()` replaced with `static_cast<int>()` and it was generally found to keep sync a bit better than mainline.

However, `std::lrint()` will round up whereas `static_cast<int>()` will truncate, so `static_cast<int>(std::floor())` will provide the same result as `std::lrint()` The main reason for changing it is that `lrint()` is apparently particularly slow on Windows. I have not tested to see if `static_cast<int>(std::floor())` is slower than `std::lrint()` in any case.